src/plugins/intel_gna/gna_plugin.cpp

```diff
@@ -1350,6 +1350,18 @@ GnaWaitStatus GNAPlugin::WaitFor(uint32_t request_idx, int64_t millisTimeout) {
                                         transpose_output_info->second);
         }
 
+        const std::set<InferenceEngine::Precision::ePrecision> supportedOutputPrecission = {
```

Comment:
Don't fully get it, the link points to `ValidateConvConcatAxis()`.

I don't agree. `ExportScores` is already overly complicated (~half of the code is dead) and could be simplified (see https://github.com/openvinotoolkit/openvino/pull/9504)
Beside IMO it is better to capture the error earlier and provide more meaningful message (exception) instead of passing wrong parameters inside the function only to throw from there.